krux_marathon_api/marathonapi.py

```diff
@@ -71,8 +52,12 @@ def connect(self, address, port):
             s.close()
```

Comment:
(Wrong place to comment since I cannot comment on unchanged line) Please do not `sys.exit(1)` in an object class. If this class were to be used in other programs besides CLI, this will cause the program to exit.

I would personally just raise the error and let `cli.py` handle it.
```python
try:
    s = socket.socket()
    s.connect((address, port))
except Exception as e:
    self.logger.error("...")
    raise
finally:
    s.close()
```